config.toml

```diff
@@ -0,0 +1,11 @@
+baseURL = "http://example.org/"
+languageCode = "en-us"
+title = "OCW Course Prototype"
+theme = "ocw-prototype"
```

Comment:
Can this be generic for now -- not necessarily bound to OCW?

Reply:
Yes, I will change the title and theme name.